app/experimenter/experiments/api/v5/types.py

```diff
@@ -237,7 +238,7 @@ class NimbusReviewType(graphene.ObjectType):
 class NimbusChangeLogType(DjangoObjectType):
     class Meta:
         model = NimbusChangeLog
-        exclude = ("id",)
+        fields = ("changed_on", "changed_by", "message", "old_status", "old_status_next")
```

Comment:
I really like as we have to explicitly mention the field's name as it increases code readability.